modules/simple-payments/simple-payments.php

```diff
@@ -155,7 +154,28 @@ function parse_shortcode( $attrs, $content = false ) {
 		return $this->output_shortcode( $data );
 	}
 
-	function ignore_shortcode() { return; }
+	function output_admin_warning( $data ) {
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
+		$css_prefix = self::$css_classname_prefix;
+
+		return "
+<div class='{$data['class']} ${css_prefix}-wrapper'>
+	<div class='${css_prefix}-product'>
+		<div class='${css_prefix}-details'>
+			<div class='${css_prefix}-purchase-message show error' id='{$data['dom_id']}-message-container'>
+				<p>" . sprintf(
+					__( 'Simple Payments is not supported by your Jetpack Plan. To learn more, and to upgrade to a supported plan, visit <a href="%s" target="_blank">these resources</a>.', 'jetpack' ),
```

Comment:
is a missing plan the only reason this could be `! $this->is_enbaled…`? the `output_admin_warning()` remains somewhat unspecific in its name but this string is very specific.

Reply:
`Simple Payments is not supported by your Jetpack Plan.`
Though, this can be output on WPCOM as well? I'd probably lose "Jetpack" for a more generic "plan".